targets/targets.json

```diff
@@ -4073,7 +4088,7 @@
             "function": "MTSCode.combine_bins_mts_dragonfly",
             "toolchains": ["GCC_ARM", "ARM_STD", "ARM_MICRO", "IAR"]
         },
-        "device_has_add": ["MPU"],
+        "device_has_add": ["MPU", "WATCHDOG"],
```

Comment:
This is not needed as target inherits from FAMILY_STM32 where WATCHDOG is already defined